css-brochure-form/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Brochure Form</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div>
+      <form>
+        <h2>LEARN MORE</h2>
+        <div class="email">
+          <label for="email">COMPANY EMAIL</label>
+        </div>
+        <div>
+          <input id="email" type="email" placeholder="sally@company.co" />
+        </div>
+        <br />
```

Comment:
Don't use `<br />` to space out elements. It's bad practice. We can utilize `margin` to space things out if needed.